### PR TITLE
Custom tabpanel selected class

### PR DESCRIPTION
**Pull request by alexanderwallin**
This PR does the following:

1. It lets you override the class name to use for currently selected `<TabPanel>`s using a new `selectedClassName` prop;
2. it only applies `display: none;` to the `<TabPanel>` if a custom `selectedClassName` prop has not been provided; and
3. adds tests.

My use case is that I need to keep the contents of hidden tabs and give them a bunch of styles in order to maintain the parent's dimensions.

### CODE REVIEW

File: src/components/TabPanel.js [hunk 1]
```diff
@@ -24,28 +25,37 @@ module.exports = React.createClass({
   getDefaultProps() {
     return {
       selected: false,
+      selectedClassName: null,
```
Comment (qisaw) [inline src/components/TabPanel.js]: `selectedClassName: 'ReactTabs__TabPanel--selected'`

File: src/components/TabPanel.js [hunk 2]
```diff
@@ -24,28 +25,37 @@ module.exports = React.createClass({
...
       tabId: null,
     };
   },
 
   render() {
-    const { className, children, selected, id, tabId, style, ...attributes } = this.props;
+    const {
+      id, tabId, selected,
+      className, selectedClassName, style, children,
+      ...attrs,
```
Comment (qisaw) [inline src/components/TabPanel.js]: why did you change this?

File: src/components/TabPanel.js [hunk 3]
```diff
@@ -24,28 +25,37 @@ module.exports = React.createClass({
...
+    } = this.props;
+
+    // Only hide using inline styles if `selectedClassName` isn't provided
```
Comment (qisaw) [inline src/components/TabPanel.js]: why?

File: src/components/TabPanel.js (hunk 2 above)
Comment (alexanderwallin) [inline src/components/TabPanel.js]: The line got 105 characters long, which didn't pass the linting.

File: src/components/TabPanel.js (hunk 3 above)
Comment (alexanderwallin) [inline src/components/TabPanel.js]: The idea is that if you specify your own custom classes, you want control over hiding/showing. With inline styles you would have to override using `!important`.

File: src/components/TabPanel.js (hunk 1 above)
Comment (alexanderwallin) [inline src/components/TabPanel.js]: This is `null` in order to satisfy inline styles condition on line 42.

Comment (alexanderwallin) [inline src/components/TabPanel.js]: Although I do think it is too much implicit logic going on.

File: src/components/TabPanel.js (hunk 3 above)
Comment (alexanderwallin) [inline src/components/TabPanel.js]: We could also just follow the design of the `Tab` component which only uses overridable classes.

The case I ran into was that I needed hidden `TabPanel`s to still take up their natural height, i.e. setting `visibility: hidden;` instead on the inline `display: none;`.
